### DISCUSSION TITLE
Sveltekit website blocks Linux visitors?

### DISCUSSION THREAD

**Discussion by galaczi** [sveltejs/kit#12666] [General]
Hi, I am using Linux Mint as my daily driver. I started to notice that some sites block me. I noticed I cannot even load Vercel's site so I thought they are the source of the block. Looks like I can load Vercel now, but still cannot load most of the sites hosted by them. 

That includes svelte.dev unfortunately. I can load the site on my Windows laptop, no problem. So my IP is not the issue here. Looks like Linux is categorized as high risk? I mean a lot of devs use Linux for obvious reasons, so this is definitely a bad idea... I would appreciate a fix. In the meantime I have to use a VPN to read the docs.

**Comment by dominikg** [comment]
might be related to https://github.com/sveltejs/svelte/issues/13170#issuecomment-2337898116

we do not block by os and i'm able to load the page on linux just fine. please check your dns cache. could take some time updating

**Comment by galaczi** [comment]
I have flushed DNS cache several times, but it didn't change anything. I think this started like 2 weeks ago already, so any IP change should have been resolved. 

My guess is Vercel is doing the blocking, not sure if it's configurable by the website owners. A lot of dev related sites don't open, and I have a feeling they are all on Vercel. I am not a paid customer there, so cannot contact them.

Edit: I can reach get.dev:
```
; <<>> DiG 9.18.28-0ubuntu0.22.04.1-Ubuntu <<>> get.dev
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 32957
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;get.dev.                       IN      A

;; ANSWER SECTION:
get.dev.                300     IN      A       216.239.32.27

;; Query time: 104 msec
;; SERVER: 127.0.0.53#53(127.0.0.53) (UDP)
```
And resolving seems to work for kit.svelte.dev too, but the site never finishes loading. Same for vercel.com.
```
; <<>> DiG 9.18.28-0ubuntu0.22.04.1-Ubuntu <<>> kit.svelte.dev
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 36956
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 65494
;; QUESTION SECTION:
;kit.svelte.dev.                        IN      A

;; ANSWER SECTION:
kit.svelte.dev.         300     IN      A       76.76.21.123

;; Query time: 16 msec
;; SERVER: 127.0.0.53#53(127.0.0.53) (UDP)
```

**Comment by galaczi** [comment]
I can also ping the site, but I am still blocked on web. A bit uncomfortable that I cannot read docs while working.
